app/assets/javascripts/discourse/app/lib/composer/toolbar.js

```diff
@@ -96,7 +145,7 @@ export class ToolbarBase {
   }
 
   addSeparator({ group = DEFAULT_GROUP, condition }) {
-    const g = this.groups.findBy("group", group);
+    const g = this.groups.find((item) => item.group === group);
```

Comment:
```suggestion
    const group = this.groups.find((item) => item.group === groupName);
```

In this case I think the argument can change to `groupName`